content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -206,7 +206,8 @@ different Kubernetes components.
 | `SuspendJob` | `false` | Alpha | 1.21 | 1.21 |
 | `SuspendJob` | `true` | Beta | 1.22 | |
 | `TopologyAwareHints` | `false` | Alpha | 1.21 | 1.22 |
-| `TopologyAwareHints` | `false` | Beta | 1.23 | |
+| `TopologyAwareHints` | `false` | Beta | 1.23 | 1.23 |
```

Comment:
https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints/ doesn't mention a step-to-enable. Was this feature previously off by default?

If so, then _ideally_ we also update https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints/ before the v1.24 release, to show that you need to enable the feature gate to have it work.

Reply:
https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints/  has been updated by @tengqm  in PR https://github.com/kubernetes/website/pull/32413/